source/comply/audit-log.rst

```diff
@@ -3,39 +3,205 @@ Audit Log v2 (Experimental) (E20)
+          {"id": 3, "name": "warn"},
+          {"id": 2, "name": "error", "stacktrace": true, "color": 31},
+          {"id": 1, "name": "fatal", "stacktrace": true},
+          {"id": 0, "name": "panic", "stacktrace": true}
+        ],
+        "maxqueuesize": 1000
+      }
+    }
+    
+.. Note::
+    Filenames for ``AdvancedLoggingConfig`` can contain an absolute filename, a relative filename, or embedded JSON.
+
+See the :download:`Advanced Logging Options Sample JSON ZIP file <../samples/advanced-logging-options-sample-json.zip>` for a sample configuration file.  
+
+Log Target Types
+~~~~~~~~~~~~~~~~
+Log target types include Console, Syslog, File, or TCP. 
+
+**Console configuration options:**
+
+.. csv-table::
+    :header: "Key", "Type", "Default", "Description"
+       
+       "out", "string", " ", "One of `stdout` or `stderr`. "
+
+**Syslog configuration options:**
+
+.. csv-table::
+    :header: "Key", "Type", "Default", "Description"
+       
+       "host", "string", " ", "IP or domain name of server to receive log records."
+       "port", "number", "", "Port number for server receiving log records."
+       "tls", "bool", "false", "When true, a TLS connection will be created."
+       "cert", "string", "  ", "Path to a cert file (.pem) to be used when establishing a TLS connection."
+       "insecure", "bool", "false", "When true, Focalbaord will accept any certificate presented by the server and any host name in that certificate. Should be used only in testing environments."
+       "tag", "string", " ", "Syslog tag field."
+
+
+**File configuration options:**
+
+.. csv-table::
+    :header: "Key", "Type", "Default", "Description"
+
+       "filename", "string", "  ", "Full path to output file."
+       "max_size", "number", "100", "Maximum size in megabytes the log file can grow before it gets rotated."
+       "max_age", "number", "0", "Maximum number of days to retain old log files based on the timestamp encoded in their filename. 0 means do not remove old log files based on age."
+       "max_backups", "number", "0", "Maximum number of old log files to retain.  0 means retain all old log files (though max_age may still cause them to get deleted.)"
+       "compress", "bool", "false", "When true, the rotated log files will be compressed using gzip."
+
+    
+**TCP configuration options:** 
+
+.. csv-table::
+    :header: "Key", "Type", "Default", "Description"
+       
+       "host", "string", "  ", "IP or domain name of server to receive log records."
+       "port", "number", " ", "Port number for server receiving log records."
+       "tls", "bool", "false", "When true, a TLS connection will be created."
+       "cert", "string", "  ", "Path to a cert file (.pem) to be used when establishing a TLS connection."
+       "insecure", "bool", "false", "When true, Mattermost will accept any certificate presented by the server and any host name in that certificate. Should be used only in testing environments."
```

Comment:
`Mattermost` is mentioned here, and `FocalBoard` above. Perhaps this section can be made generic:

```suggestion
       "insecure", "bool", "false", "When true, any certificate will be accepted and any host name in that certificate. Should be used only in testing environments."
```